tests/Type/Doctrine/Query/QueryResultTypeWalkerTest.php

```diff
@@ -75,6 +74,10 @@ public static function getAdditionalConfigFiles(): array
 
 	public static function setUpBeforeClass(): void
 	{
+		if (PHP_VERSION_ID < 80100) {
+			self::markTestSkipped('Test uses asserts without stringification, those work only with PHP >= 8.1. Stringification is tested in platform test.');
```

Comment:
So this entire test is skipped below PHP 8.1? I don't understand the reason and I'd prefer if the test wasn't skipped.

Reply:
Adjusted the asserts, now it run everywhere.